pkg/disttask/framework/scheduler/interface.go

```diff
@@ -30,6 +30,7 @@ type TaskManager interface {
 	// The returned tasks are sorted by task order, see proto.Task, and only contains
 	// some fields, see row2TaskBasic.
 	GetTopUnfinishedTasks(ctx context.Context) ([]*proto.Task, error)
+	GetAllSubtasks(ctx context.Context) ([]*proto.Subtask, error)
```

Comment:
Maybe we should add a struct like `proto.SubtaskBase`, which only contains part of columns. Or add a comment about some fields of `proto.Subtask` are empty.

Reply:
+1

Reply:
SGTM, But it is better to update in another pr. I can do it if we need.